From phil@ipom.com  Thu Dec 18 15:34:39 2008
From: Oliver Neukum <oliver@neukum.org>
Date: Fri, 12 Dec 2008 11:01:45 +0100
Subject: USB: storage: extend unusual range for 067b:3507
To: Greg Kroah-Hartman <gregkh@suse.de>, usb-storage@lists.one-eyed-alien.net, linux-usb@vger.kernel.org
Message-ID: <20081212100145.GN20335@ipom.com>
Content-Disposition: inline

From: Oliver Neukum <oliver@neukum.org>

This device has been released in a new revision which is still buggy.

Signed-off-by: Oliver Neukum <oneukum@suse.de>
Signed-off-by: Phil Dibowitz <phil@ipom.com>
Cc: stable <stable@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/unusual_devs.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/storage/unusual_devs.h
+++ b/drivers/usb/storage/unusual_devs.h
@@ -1040,7 +1040,7 @@ UNUSUAL_DEV( 0x067b, 0x2507, 0x0100, 0x0
 		US_FL_FIX_CAPACITY | US_FL_GO_SLOW ),
 
 /* Reported by Alex Butcher <alex.butcher@assursys.co.uk> */
-UNUSUAL_DEV( 0x067b, 0x3507, 0x0001, 0x0001,
+UNUSUAL_DEV( 0x067b, 0x3507, 0x0001, 0x0101,
 		"Prolific Technology Inc.",
 		"ATAPI-6 Bridge Controller",
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
